albam/engines/mtfw/structs/arc.ksy

```diff
@@ -7,21 +8,27 @@ meta:
 
 
 seq:
-  - {id: ident, contents: [0x41, 0x52, 0x43, 0x00]}
-  - {id: version, type: s2}
-  - {id: num_files, type: s2}
-  - {id: file_entries, type: file_entry, repeat: expr, repeat-expr: num_files}
-  # TODO: padding
+  - {id: header, type: arc_header}
+  - {id: files, type: file_entry, repeat: expr, repeat-expr: _root.header.num_files}
+  - {id: padding, size: 32760 - _root.header.num_files * 80}
```

Comment:
I found more than one case in RE5 with arcs wit more than 410 file entries, where this will break.
The correct algorithm to calculate the padding should be:
`-(_root.header.num_files * 80) % 32760`

This will give a padding `40` for 409 files, and an additional 32720 bytes of padding for 410 files